components/layout/wrapper.rs

```diff
@@ -470,6 +470,12 @@ impl<'le> ::selectors::Element for LayoutElement<'le> {
         self.element.get_event_state_for_layout().contains(element::IN_FOCUS_STATE)
     }
 
+    // FIXME(#7720)
+    #[inline]
```

Comment:
No need for the hint; LLVM can figure this out on its own and inline hints slow compile times in most cases 

(brson did a study on the tradeoffs of `#[inline]`, in almost all cases it didn't help)